test/parallel/test-child-process-bad-stdio.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+// Flags: --expose_internals
+const common = require('../common');
+const assert = require('assert');
+const cp = require('child_process');
+
+if (process.argv[2] === 'child') {
+  setTimeout(() => {}, common.platformTimeout(1000));
```

Comment:
A `setImmediate()` is likely a safe. I just wanted to make sure that the spawned process didn't complete before the 1ms timeout in the third scenario.